src/AccountManagement.ts

```diff
@@ -131,9 +129,9 @@ export class AccountManagement {
   /**
    * Send an Authorization Request to an Account to subsidize its transaction fees
    *
-   * @note this will create an AuthorizationRequest which has to be accepted by the beneficiary Account.
-   *   An Account or Identity can fetch its pending Authorization Requests by calling {@link Authorizations.getReceived}.
-   *   Also, an Account or Identity can directly fetch the details of an Authorization Request by calling {@link Authorizations.getOne}
+   * @note this will create an {@link AuthorizationRequest Authorization Request} which has to be accepted by the `beneficiary` Account.
```

Comment:
```suggestion
   * @note this will create an {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `beneficiary` Account.
```